connector_sage/models/hr_employee/binding.py

```diff
@@ -0,0 +1,50 @@
+# Copyright NuoBiT Solutions - Eric Antones <eantones@nuobit.com>
+# Copyright NuoBiT Solutions - Kilian Niubo <kniubo@nuobit.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models
+
+
+class HrEmployee(models.Model):
```

Comment:
@KNVx this class has no "_description" field and there's other PRs with the same problem. Please run each of your PR's in a test environment and check the log warnings in order to know what to fix. Thanks